spec/support/factories.rb

```diff
@@ -15,6 +15,18 @@
     author_id '1'
   end
 
+  factory :dynamic_book, class: Book do
+    sequence(:title) do |i|
+      i = i.to_s.rjust(2, "0") if i < 10
```

Comment:
Is there a specific reason to create "Book Title01"? It just looked a little unusual.